ybacklight

```diff
@@ -64,7 +64,7 @@ sub get_percentage {
     my ($driver) = @_;
     my $bri = get_numeric_attribute($driver, "brightness");
     my $bri_max = get_numeric_attribute($driver, "max_brightness");
-    my $pct = int($bri/$bri_max*100);
+    my $pct = ($bri/$bri_max*100);
```

Comment:
Will do.  I left them because I noticed that, e.g.:
`my $variable = $value/100;`
wouldn't "pick up" the `/100` bit, and instead store `$value` directly to `$variable`.  Shows my lack of Perl experience, I image.  The solution was to wrap the expression in parenthesis:
`my $variable = ($value/100);`